controllers/result.go

```diff
@@ -74,7 +74,9 @@ func CreateResult(ctx context.Context, w http.ResponseWriter, r *http.Request) (
 		return http.StatusInternalServerError, err
 	}
 
-	result := model.Result{Challenge: key, StartTimes: make([]time.Time, len(challenge.Tasks))}
+	result := model.Result{Challenge: key,
+		StartTimes:       make([]time.Time, len(challenge.Tasks)),
+		FinalSubmissions: make([]*datastore.Key, len(challenge.Tasks))}
```

Comment:
I think it should be either everything in one line, or every property on it's own line:

```go
result := model.Result{
	Challenge:        key,
	StartTimes:       make([]time.Time, len(challenge.Tasks)),
	FinalSubmissions: make([]*datastore.Key, len(challenge.Tasks)),
}
```